middle.mkd

```diff
@@ -189,6 +189,30 @@ Implementations SHOULD NOT extend positions beyond 3 elements. Parsers
 MAY ignore additional elements. Interpretation and meaning of additional
 elements is beyond the scope of this specification.
 
+A line between two positions is a straight Cartesian line, the shortest
```

Comment:
Normative, because allowing either leads to that awkward situation @dirkx brought up: hemisphere-spanning lines with radically different lengths, trajectories, and bounding boxes depending on whether they are straight Cartesian lines or geodesics.